### PR TITLE
chore(tslint): fix tslint config & errors

### PR DESCRIPTION
**Pull request by manoldonev**
Fix tslint.json syntax for some rules and the tslint errors uncovered by that fix.

### CODE REVIEW

File: build/tslint.json [hunk 1]
```diff
@@ -3,56 +3,83 @@
...
+        "one-line": [
+            false,
             "check-open-brace",
             "check-catch",
             "check-else",
             "check-whitespace"
         ],
-        "quotemark": [false, "double"],
+        "quotemark": [
+            false,
```
Comment (sis0k0) [inline build/tslint.json]: Shouldn't this be `true` if we want to force `double` quotes?

Comment (manoldonev) [reply build/tslint.json]: @sis0k0 I did not disable this rule originally but it is now enabled and the errors fixed :).

**Comment by manoldonev** [comment]
@sis0k0 enabled additionally `arrow-return-shorthand` and `whitespace` rules.

**Comment by manoldonev** [comment]
test

**Pull request merged**